spec/features/user_can_update_qty_of_item_in_cart_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.feature "visitor can update quantity of items in cart" do
+  scenario "they can add to quantity of item in cart" do
+    db_repo = FactoryJordan.new
+    db_repo.create_all
+    item_1 = db_repo.items[0][0]
+
+    visit '/items'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.